src/brijesh/codingexams/Exam1.java

```diff
@@ -0,0 +1,46 @@
+package brijesh.codingexams;
+
+import java.awt.DisplayMode;
+import java.util.Scanner;
+
+import javax.xml.ws.Dispatch;
+
+public class Exam1 {
+
+	String getMaxLengthName(String[] arr) {
+		int max=arr[0].length();
+		String maxString=arr[0];
+		for(int i=1; i<arr.length;i++) {
+			if(arr[i].length()>max) {
```

Comment:
max variable can be removed. 
if(arr[i].length() > maString.length())

Reply:
Yes Thanks Got it.